Makefile

```diff
@@ -43,7 +43,7 @@ run-dev: load-env
 	RUST_LOG=trace cargo run --bin kakarot-rpc
 
 install-katana:
-	cargo install --git https://github.com/dojoengine/dojo --locked --tag v0.7.3 katana
+	cargo install --git https://github.com/dojoengine/dojo --locked --tag v1.0.0-alpha.0 katana
```

Comment:
Even `v1.0.0-alpha.1` is ok no? since cairo 2.7 is in `v1.0.0-alpha.2`

Reply:
I guess it's ok, I just went with the same version as on the Kakarot repo